### ISSUE TITLE
Organization Personnel vs Career history

### ISSUE THREAD

**Issue by evz** [security-force-monitor/sfm-cms#511]
As far as I can tell, the [Organization personnel](https://github.com/security-force-monitor/sfm-cms/wiki/What-raw-data-do-we-need-to-download-in-bulk-from-WhoWasInCommand.com%3F#organizations-personnel) download is pretty much the same as the [Person career history](https://github.com/security-force-monitor/sfm-cms/wiki/What-raw-data-do-we-need-to-download-in-bulk-from-WhoWasInCommand.com%3F#person-full-career-histories) download except we're leaving out some of the fields concerning the organization in the latter. Is that accurate? Is there an opportunity to consolidate those?

**Comment by evz** [comment]
@tlongers & @tonysecurityforcemonitor  ^

**Comment by tlongers** [comment]
Yep, the difference is only really in column ordering; Content-wise, you're right, they can be consolidated.  Extend the organizational personal download to include the full person profiles rather than the trimmed down one.

@tonysecurityforcemonitor Am I right on that?

**Comment by tonysecurityforcemonitor** [comment]
@evz @tlongers - that makes sense - as long as it makes sense to the downloader that it will display the full profile of every person related to an organization.
